pages/blog/release-notes-2.4.0.md

```diff
@@ -0,0 +1,136 @@
+---
+title: AsyncAPI Spec 2.4.0 Release Notes
+date: 2022-04-25T19:00:00+01:00
+type: Communication
+tags:
+  - Specification
+  - Release Notes
+cover: /img/posts/release-notes-2.4.0/cover.webp
+authors:
+  - name: Sergio Moya
+    photo: /img/avatars/smoya.webp
+    link: https://twitter.com/smoyac
+    byline: Pilot at AsyncAPI Airlines
+excerpt: 'AsyncAPI 2.4 is now released. This brings really helpful additions, such as the new `messageId' field, Server Variables reusability, and security at Operation level'
+featured: true
+---
+
+The new version of the AsyncAPI specification - 2.4.0 - is now available.
+
+> This is a minor release, and it doesn't bring any breaking changes. You can switch to it by modifying the following value in your AsyncAPI file `asyncapi: '2.3.0'` into `asyncapi: '2.4.0'`
+
+## Message uniqueness thanks to the new messageId field
```

Comment:
For some reason, I dropped the notes for this feature. Now I recovered them.

cc @WaleedAshraf, in case you want to give an eye.
cc @derberg @alequetzalli as this will be new for you to review.